### ISSUE TITLE
How does it get Electrolux Wellbeing to ask for credentials/API key again?

### ISSUE THREAD

**Issue by ryssel** [JohNan/homeassistant-wellbeing#127]
How does it get Electrolux Wellbeing to ask for credentials/API key again?
I am running version 1.2.3 but it says v0.0.0 in Integrations
I get this error in my logfile: [pyelectroluxgroup.auth] Failed to get access token: Not enough segments
![image](https://github.com/user-attachments/assets/ef41d207-cfaf-423a-af9f-98156fd7cd6f)
![image](https://github.com/user-attachments/assets/d2ae15c0-e2de-4ed5-8acd-6fa84d3cb6e9)

**Comment by patrickis74** [comment]
Test this, 
To use this integration you need to login in to https://developer.electrolux.one/ using your Electrolux account details and generate an API key, Access Token and Refresh Token that is needed in the configuration.

**Comment by ryssel** [comment]
I have created an Api Key, an Access Token, and a Refresh Token, but the integration does not ask me to authenticate again.
Where is the previous authentication values for Electrolux Wellbeing saved? Can I delete them and it will ask me again?

**Comment by JohNan** [comment]
It's saved in the config entry. Try remove the integration and add it again.

**Comment by ryssel** [comment]
Removing and adding the integration again solved the problem, but as expected I lost all historical data.

**Comment by d3zit** [comment]
When re-added the sensor name template also appears to be different. It would be great to add the possibility force the  API / Token request again without removing and readding the sensor

**Comment by sneabs** [comment]
Indeed, I ran into the same issue after a restore from a backup where the integration lost its credentials to use the API. These credential seems to not contained in backup or no longer usable
